README.md

```diff
@@ -22,8 +22,22 @@ For this reason, create a service account and the appropriate cluster role bindi
 kubectl apply -f appsody-service-account.yaml
 kubectl apply -f appsody-cluster-role-binding.yaml
 ```
+The previous commands set up a service account called `appsody-sa` and grant the `cluster-admin` role to it. The pipeline you are going to create uses this service account. 
 
-2) Now, create the pipeline task and the pipeline definition. We have  a simple pipeline, with a single task that performs the various steps of building and deploying the project:
+2) Make the Docker registry credentials available to the pipeline by adding your docker secret to the `appsody-sa` service account. This can be accomplished by editing the service account, using the following command:
+```
+kubectl edit serviceaccount appsody-sa
+```
+An editor opens up. Add your secret to the list of secrets, as shown in the example below: 
+```
+...
+secrets:
+- name: appsody-sa-token-ldzbq
+- name: my-docker-secret
+```
+Save the changes. 
+
+2) Now, create the pipeline task and the pipeline definition. We have a simple pipeline, with a single task that performs the various steps of building and deploying the project:
```

Comment:
Line 49:

For this reason, you need to edit the `appsody-pipeline-resource.yaml`
----
Looks like it should be 'appsody-pipeline-resourceS.yaml'

Reply:
Fixed - thanks.